checker/src/org/checkerframework/checker/samelen/SameLenUtils.java

```diff
@@ -0,0 +1,20 @@
+package org.checkerframework.checker.samelen;
+
+import javax.lang.model.element.AnnotationMirror;
+import org.checkerframework.checker.upperbound.qual.*;
+import org.checkerframework.javacutil.AnnotationUtils;
+
+public class SameLenUtils {
+
+    /**
+     * Used to get the list of array names that an annotation applies to. Can return null if the
+     * list would be empty.
+     */
+    public static String[] getValue(AnnotationMirror anno) {
```

Comment:
Is there any reason this returns an array rather than a list?  The ATF has a lot of code converting arrays to/from lists because of this return type.